.travis.yml

```diff
@@ -1,23 +1,16 @@
 sudo: false
 language: ruby
 rvm:
-  #- 2.3.3
+  - 2.3.6
   - 2.4.2
 
 env:
   global:
     - BROWSER=chrome
   matrix:
-    - DB=mysql
     - DB=postgres
-    - DB=sqlite
-
-matrix:
-  exclude:
-    - rvm: 2.3.3
```

Comment:
Probably better to just update this exclude to 2.3.6, ditto below for the unlikely chance we bring in mysql/sqlite regression